src/main/java/seedu/address/model/moduleclass/UniqueModuleClassList.java

```diff
@@ -116,6 +117,15 @@ public void remove(ModuleClass toRemove) throws NullPointerException, ModuleClas
         }
     }
 
+    /**
+     * Removes all {@code studentUuids} from each and every {@code ModuleClass} in the list.
+     */
```

Comment:
Would it be good to specify which list?
```suggestion
    /**
     * Removes all {@code studentUuids} from each and every {@code ModuleClass} in the class list.
     */
```

Reply:
Perhaps `{@code Student UUID}s` would be more accurate since `{@code studentUuids}` refers to the list itself rather than its contents.
```suggestion
    /**
     * Removes all {@code Student UUID}s from every {@code ModuleClass} in the class list.
     */
```